state/container_test.go

```diff
@@ -167,38 +167,6 @@ func Test_container_Data(t *testing.T) {
 	}
 }
 
-func Test_container_String(t *testing.T) {
```

Comment:
Minor: why are you removing this? This method (mount stringer) is used in various places which may be expecting a specific output format. Definitely not a critical UT, but it doesn't bother us either.

Reply:
I removed it because the test was failing (due to a change I did earlier in sysbox-fs logging), and looking at the function, I saw it was not worth keeping/maintaining since it only tests the stringer function, which simply logs something.